src/data/set/basic.lean

```diff
@@ -922,6 +925,9 @@ ext $ assume x, ⟨assume ⟨hx, _⟩, hx, assume h, ⟨h, not_false⟩⟩
 theorem diff_diff {u : set α} : s \ t \ u = s \ (t ∪ u) :=
 ext $ by simp [not_or_distrib, and.comm, and.left_comm]
 
+lemma diff_diff_comm {s t u : set α} : s \ t \ u = s \ u \ t :=
```

Comment:
I know parentheses are not necessary for Lean here, but could you add them for the human reader? (i.e., I don't know if you are talking about `s \ (t \ u)` or `(s \ t) \ u` without opening vscode)

Reply:
It's talking about the unique way that makes the lemma true in general :)
I added the parentheses.